File: Domain/Location.cs
using System;

namespace Domain
{
    public class Location
    {
        public Guid Id { get; set; }
        public string FormattedLocation { get; set; }
        public string LocationString { get; set; }
        public double Lat { get; set; }
        public double Lng { get; set; }
    }
}

File: client/src/App/Interfaces/Location.ts
export interface Location
{
    id?: string;
    formattedLocation: string;
    locationString: string;
    lat: number;
    lng: number;
}

export class Location
{
    constructor(lat= 0, lng=0, formattedLocation="", locationString="", id=undefined)
    {
        this.id = id;
        this.lat = lat;
        this.lng = lng;
        this.locationString = locationString;
        this.formattedLocation = formattedLocation;
        
    }
}